# HG changeset patch
# Parent 0b1d1198f1fbc75e812f99901743f39903c56c57
# User Ben Kelly <ben@wanderview.com>
Fix w3c test to require Request bodyUsed flag to be set.


diff --git a/testing/web-platform/tests/cache/resources/cache-put-worker.js b/testing/web-platform/tests/cache/resources/cache-put-worker.js
--- a/testing/web-platform/tests/cache/resources/cache-put-worker.js
+++ b/testing/web-platform/tests/cache/resources/cache-put-worker.js
@@ -60,18 +60,18 @@ cache_test(function(cache) {
         body: 'Hello'
       });
     var response = new_test_response();
     assert_false(request.bodyUsed,
                  '[https://fetch.spec.whatwg.org/#dom-body-bodyused] ' +
                  'Request.bodyUsed should be initially false.');
     return cache.put(request, response)
       .then(function() {
-          assert_false(request.bodyUsed,
-                       'Cache.put should not consume Request body.');
+          assert_true(request.bodyUsed,
+                       'Cache.put should consume Request body.');
         })
       .then(function() {
           return cache.match(request);
         })
       .then(function(result) {
           assert_object_equals(result, response,
                                'Cache.put should store response body.');
         });
